.github/workflows/ampli-merge-check.android.yml

```diff
@@ -19,10 +19,15 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v2
 
-      - name: (Android Kotlin) Check the Data branch is merged before merging the Git branch
-        working-directory: ./android/kotlin/AmpliApp
+      - name: (Android Kotlin V1) Check the Data branch is merged before merging the Git branch
+        working-directory: ./android/kotlin/v1/AmpliApp
         run: ampli status --is-merged -t ${{secrets.AMPLI_TOKEN}}
 
+      - name: (Android Kotlin V1) Check the Data branch is merged before merging the Git branch
+        working-directory: ./android/kotlin/v2/AmpliApp
```

Comment:
This test is failing since the new runtime/seed data is not on stage. We will need to update the stage DB with the new data to fix.

Reply:
Separated android ampli merge check v1 and v2 into different Github actions.